autograder/handgrading/views/comment_views.py

```diff
@@ -21,10 +21,9 @@ def has_object_permission(self, request, view, obj):
             is_admin = course.is_admin(request.user)
             is_staff = course.is_staff(request.user)
             is_handgrader = course.is_handgrader(request.user)
-            read_only = request.method in permissions.SAFE_METHODS
 
-            return is_admin or (is_handgrader and can_edit) or (read_only and (is_staff
-                                                                               or is_handgrader))
+            return is_admin or (can_edit and (is_staff or is_handgrader))
```

Comment:
I'm fairly certain this change would make it so that handgraders can't send get requests for comments, which isn't something we want. See my earlier comment about what value of "handgraders_can_leave_comments" the tests use.